include/seqan3/argument_parser/detail/format_parse.hpp

```diff
@@ -323,8 +323,11 @@ class format_parse : public format_base
 
         if (auto it = map.find(in); it == map.end())
         {
+            auto keys_view = map | std::views::keys;
```

Comment:
```suggestion
            auto keys_view = map | std::views::keys | std::views::common;
```

Reply:
(That's the one I always forget xD)

Reply:
Maybe we should do the second version and just copy the map as is and sort that.

And extract the keys afterwards `". Please use one of: ", keys | std::views::keys))};`

Reply:
If i'd copied the pairs, i would need to fully specify the `std::vector`, maps return `std::pair<const Key, Value>`, but we would need `std::pair<Key, Value>` to be able to sort

Reply:
Actually, gcc7 cannot handle the ctad, so I'll need to define the vector anyways and might as well just copy the pairs

Reply:
👍🏽